Mathlib/Data/Sum/Basic.lean

```diff
@@ -123,11 +123,11 @@ variable {x y : Sum α β}
   cases x <;> simp only [getRight?, isLeft, eq_self_iff_true]
 #align sum.get_right_eq_none_iff Sum.getRight?_eq_none_iff
 
-@[simp] lemma getLeft?_eq_some_iff {a : α} : x.getLeft? = a ↔ x = inl a := by
+@[simp] theorem getLeft?_eq_some_iff {a : α} : x.getLeft? = a ↔ x = inl a := by
```

Comment:
why this change?

Reply:
Oh, it will have been an auto-replace. I was told we don't use lemma?

Reply:
Really I should make this change in my other PR.

Reply:
If you have a link to that I'd like to read the discussion.
My understanding was simply that mathport didn't use lemma so there are mostly "theorem"s in mathlib, but `mathlib` does add the lemma macro so it seems its up to personal preference to use it or not.
I just worry that by changing it we get into a weird state where some people are adding lemmas and other people changing them to theorems and its a lot of inessential changes. So it would be good to have either lemma banned across mathlib and fixed all at once or leave things as they are

Reply:
Unfortunately it was on Kevin's discord. Though I also had such a comment in another PR recently...

I don't think there's any clear guidance currently. I don't understand what the criteria are.

Reply:
Yeah in that case I think its better to just leave existing lines as they are then, until there becomes a clear consensus

Reply:
It seems we're moving to a situation where most declarations are `lemma` and important ones are `theorem`.

Reply:
I'm not going to change my PR for this though. I already changed it back.